include/mbedtls/config_adjust_legacy_from_psa.h

```diff
@@ -542,6 +542,49 @@
 #endif /* !MBEDTLS_PSA_ACCEL_ALG_CHACHA20_POLY1305 */
 #endif /* PSA_WANT_ALG_CHACHA20_POLY1305 */
 
+/* Ensure that the PSA's supported curves (PSA_WANT_ECC_xxx) are always a
```

Comment:
This is in the wrong file.

This is fixed in a later commit in https://github.com/Mbed-TLS/mbedtls/pull/8075.